data/dn-group.yaml

```diff
@@ -0,0 +1,6 @@
+DNGroup1:
```

Comment:
I think we just need to send some hardcoded response and don't have to have these 2 files defined. We can many of the changes this way. avoid changes to 3 files. What do you think?

Reply:
As discussed removed the yaml files and just send the empty object as a response for dn-group and virtual queue.